.ruby-version

```diff
@@ -1 +0,0 @@
-2.1.3
```

Comment:
[Using the Gemfile works for RVM](https://rvm.io/workflow/projects#supported-files), but fair enough in case someone else might be using [rbenv](https://github.com/sstephenson/rbenv#choosing-the-ruby-version).